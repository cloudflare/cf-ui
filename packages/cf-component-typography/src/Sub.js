import React from 'react';

import PropTypes from 'prop-types';

class Sub extends React.Component {
  render() {
    return (
      <sub className="cf-sub">
        {this.props.children}
      </sub>
    );
  }
}

Sub.propTypes = {
  children: PropTypes.node
};

export default Sub;
