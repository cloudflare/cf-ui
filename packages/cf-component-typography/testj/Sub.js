import React from 'react';
import renderer from 'react-test-renderer';
import Sub from '../src/Sub';

describe('Sub', () => {
  test('should render', () => {
    const component = renderer.create(<Sub>Sub</Sub>);
    expect(component.toJSON()).toMatchSnapshot();
  });
});
