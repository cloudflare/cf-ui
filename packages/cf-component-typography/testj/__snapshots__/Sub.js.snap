// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Sub should render 1`] = `
<sub
  className="cf-sub"
>
  Sub
</sub>
`;
